src/chain.h

```diff
@@ -320,6 +324,7 @@ class CDiskBlockIndex : public CBlockIndex
             READWRITE(VARINT(nDataPos));
         if (nStatus & BLOCK_HAVE_UNDO)
             READWRITE(VARINT(nUndoPos));
+        READWRITE(hashAnchor);
```

Comment:
IIRC a `-reindex` launch. We should double-check this and then specify it in the release blog post.